specs/applicationdata/ApplicationData.md

```diff
@@ -0,0 +1,432 @@
+        String Name { get; };
+
+        /// Return the type of the app data store that is associated with the settings container.
+        /// @see https://learn.microsoft.com/uwp/api/windows.storage.applicationdatacontainer.locality
+        ApplicationDataLocality Locality { get; };
+
+        /// Return an object that represents the settings in the settings container.
+        /// @see https://learn.microsoft.com/uwp/api/windows.storage.applicationdatacontainer.values
+        Windows.Foundation.Collections.IPropertySet Values { get; };
+
+        /// Create or open the specified settings container in the settings container.
+        /// @see https://learn.microsoft.com/uwp/api/windows.storage.applicationdatacontainer.createcontainer
+        ApplicationDataContainer CreateContainer(String name, ApplicationDataCreateDisposition disposition);
+
+        /// Delete the specified settings container, its subcontainers, and all settings in the hierarchy.
+        /// @see https://learn.microsoft.com/uwp/api/windows.storage.applicationdatacontainer.deletecontainer
+        void DeleteContainer(String name);
+    };
+
+    /// Provides access to the application data store.
+    /// @see https://learn.microsoft.com/uwp/api/windows.storage.applicationdata
+    [contract(ApplicationDataContract, 1)]
+    runtimeclass ApplicationData : Windows.Foundation.IClosable
+    {
+        /// Get an instance of ApplicationData for the current user.
+        /// @note Requires the current process has package identity.
+        /// @note This is equivalent to Windows.Storage.ApplicationData.Current if running in an AppContainer.
+        /// @note This is equivalent to Windows.Storage.ApplicationDataManager.CreateForPackageFamily(GetCurrentPackageFamilyName()) if not running in an AppContainer.
+        static ApplicationData GetDefault();
+
+        /// Get an instance of ApplicationData for the specified user.
+        /// @note This is equivalent to Windows.Storage.ApplicationDataManager.GetForUserAsync()
+        /// @see https://learn.microsoft.com/uwp/api/windows.storage.applicationdata.getforuserasync
+        static ApplicationData GetForUser(Windows.System.User user);
+
+        /// Get an instance of ApplicationData for the specified package family for the current user.
+        /// @note This is equivalent to Windows.Storage.ApplicationDataManager.CreateForPackageFamily()
+        /// @see https://learn.microsoft.com/uwp/api/windows.management.core.applicationdatamanager.createforpackagefamily
+        static ApplicationData GetForPackageFamily(String packageFamilyName);
+
+        /// Get an instance of ApplicationData for the specified unpackaged app for the current user.
+        static ApplicationData GetForUnpackaged(String publisher, String name);
+
+        /// Return true if the package family supports the machine data store.
+        Boolean IsMachinePathSupported();
```

Comment:
Should this be a property instead of a method?

Reply:
Good point!

RECOMMEND: Make IsMachinePathSupported a get property